test/test_utils/sagemaker.py

```diff
@@ -268,3 +267,15 @@ def execute_sagemaker_remote_tests(image):
         with context.prefix(f"source {tag}/bin/activate"):
             context.run("pip install -r requirements.txt", warn=True)
             context.run(pytest_command)
+
+
+def generate_empty_report(report):
+    """
+    Creating an empty reports for EIA and TF PY2 Inference Images
```

Comment:
Can we not explicitly define the caller's conditions in the function?  You can instead say "function to generate empty junitxml report if no tests are run"

Reply:
Made the changes in new revision.